config/config.yml

```diff
@@ -62,7 +62,8 @@ build_composer: true
 drupal_composer_path: false
 drupal_composer_install_base_dir: "/var/www"
 drupal_composer_install_dir: "{{ drupal_composer_install_base_dir }}/drupalvm"
-drupal_composer_site_install_dir: "{{ drupal_composer_install_base_dir }}/sites"
+drupal_composer_site_host_dir: "~/Sites/projects"
+drupal_composer_site_install_dir: "{{ drupal_composer_install_base_dir }}/projects"
```

Comment:
… because two host directories are mounted 

1. the base directory of the Drupal VM;
2. the projects/sites directory